ci-operator/step-registry/workers-rhel/repos/workers-rhel-repos-commands.sh

```diff
@@ -35,6 +37,8 @@ cat > prep.yaml <<-'EOF'
   vars:
     kubeconfig_path: "{{ lookup('env', 'KUBECONFIG') }}"
     ops_mirror_path: "{{ lookup('env', 'OPS_MIRROR_KEY') }}"
+    platform_version: "{{ lookup('env', 'PLATFORM_VERSION') }}"
+    major_platform_version: "{{ platform_version[:1] }}"
```

Comment:
It appears `platform_version` is used to extract the major version only - perhaps we should be passing `7`/`8` instead of `7.9`/`8.4` here?

Reply:
The full platform_version is used in the aws provisioning playbook.  I can possibly look at possibly changing how we search for the correct AMI to use.

https://github.com/openshift/release/pull/19190/files/66ee745afe2ce328c1aca90ded35e6b836d2255e#diff-32072cf9dca6e77ad0d87041629c4a45d24acb4bf6becc40947250d0a48c2c58R45

Reply:
Ah, I see, thanks